codespell_lib/data/dictionary.txt

```diff
@@ -10514,6 +10519,8 @@ edxpected->expected
 eearly->early
 eeeprom->EEPROM
 eescription->description
+eevery->every
```

Comment:
Likewise more rows for -rywhere, -rything

Reply:
added.